### PR TITLE
refactor: rename deposit record

### PR DESCRIPTION
**Pull request by davidprepo**
change CollateralDepositRecord.sol to DepositRecord.sol

### CODE REVIEW

**Comment by ramenforbreakfast** [comment]
Will continue reviewing this later, but seems like we can't change that `developer.js` file or it will break the frontend doc build due to the doc still being named `CollateralDepositRecord`. We will need to change these docs anyway later so lets leave that out of this PR.

File: apps/smart-contracts/core/contracts/interfaces/IDepositRecord.sol [hunk 1]
```diff
@@ -52,7 +52,7 @@ interface ICollateralDepositRecord {
   function recordWithdrawal(uint256 amount) external;
 
   /**
-   * @notice Sets the global cap on assets backing Collateral in circulation.
+   * @notice Sets the global deposit cap on assets.
```
Comment (XavierEkkel) [inline apps/smart-contracts/core/contracts/interfaces/IDepositRecord.sol:55]:
```suggestion
   * @notice Sets the global net deposit cap.
```

**Review by XavierEkkel — APPROVED** [review]
LGTM after Michael's comment and after ensuring the final PR compiles + passes tests (can look in the `Checks` tab of the PR)

**Review by ramenforbreakfast — APPROVED** [review]
LGTM, but you need to rename the deployment script to `DepositRecord.ts`, validated tests are working.

File: apps/smart-contracts/core/contracts/interfaces/IDepositRecord.sol (hunk 1 above)
Comment (davidprepo) [reply apps/smart-contracts/core/contracts/interfaces/IDepositRecord.sol]: Fixed in the latest commit.

**Pull request merged**